docs/configuration_file.md

```diff
@@ -14,6 +14,8 @@ key=value
 
 There is section `[general]` which contains global configuration for all instances. You can put more sections to your config which represents multiple instances of OSBS (and then you can refer to them with command line option `--instance`).
 
+It is also possible to describe platforms in platform sections, which have the form `[platform:ARCH]`, where `ARCH` is the platform name, such as `x86_64` or `aarch64`. Platform sections are not interpreted as OSBS instances, but descriptions to be applied to OSBS instances running on the given platform.
```

Comment:
Minor: could remove commas before "where" and "such as".

Reply:
done